Dockerfiles/midstream/run_tests.py

```diff
@@ -6,42 +6,66 @@
 import subprocess
 from os import path
 
+# global variables set 
+OPERATOR_DIR="/tmp/operator_dir/"
+OPERATOR_WORK_DIR="/tmp/test_operator_work_dir/"
+WORK_DIR="/tmp/output/"
+PLAYBOOKS_DIR="/tmp/operator-test-playbooks/"
+UMOCI_BIN_PATH="/usr/local/bin/umoci"
 
 class RunMidstreamCVPTests():
 
     def setUp(self):
-        self.operator_dir = os.getenv('OPERATOR_DIR',
-                                      "/project/operator_dir/")
-        self.operator_work_dir = os.getenv('OPERATOR_WORK_DIR',
-                                           "/project/test_operator_work_dir/")
-        self.work_dir = os.getenv('WORK_DIR',
-                                  "/project/output/")
-        self.playbooks_dir = os.getenv('PLAYBOOKS_DIR',
-                                       "/project/operator-test-playbooks/")
         self.image_to_test = os.getenv('IMAGE_TO_TEST')
-        self.umoci_bin_path = os.getenv('UMOCI_BIN_PATH',
-                                        '/usr/local/bin/umoci')
+        self.verbosity = int(os.getenv('VERBOSITY', '1'))*"v"
 
     def run_extract_operator_bundle(self):
-        exec_cmd = "ansible-playbook -i localhost, -c local -v operator-test-playbooks/extract-operator-bundle.yml \
+        exec_cmd = "ansible-playbook -i localhost, -c local -{verbosity} {playbook_dir}/extract-operator-bundle.yml \
                     -e 'bundle_image={bundle_image}' \
                     -e 'operator_work_dir={operator_work_dir}' \
-                    -e 'umoci_bin_path={umoci_bin_path}'".format(
-                                                    operator_dir=self.operator_dir,
-                                                    operator_work_dir=self.operator_work_dir,
+                    -e 'umoci_bin_path={umoci_bin_path}' \
+                    -e 'work_dir={work_dir}'".format(
+                                                    operator_dir=OPERATOR_DIR,
+                                                    playbook_dir=PLAYBOOKS_DIR,
+                                                    operator_work_dir=OPERATOR_WORK_DIR,
                                                     bundle_image=self.image_to_test,
-                                                    umoci_bin_path=self.umoci_bin_path)
+                                                    umoci_bin_path=UMOCI_BIN_PATH,
+                                                    verbosity=self.verbosity,
+                                                    work_dir=WORK_DIR)
+        my_env = os.environ.copy()
+        my_env["ANSIBLE_CONFIG"] = "/tmp/"
+        my_env["ANSIBLE_LOCAL_TEMP"] = "/tmp/" + my_env["PATH"]
+        print(exec_cmd)
+        result = ""
+        with subprocess.Popen(exec_cmd, stdout=subprocess.PIPE, stderr=None, shell=True, env=my_env) as process:
+           output = process.communicate()[0].decode("utf-8")
+           print(output)
+           result = output
```

Comment:
fedora 33 has python 3.9 which has much simpler way for text output from scripts.
You can use either parameter `universal_newlines` or `text`

https://docs.python.org/3.9/library/subprocess.html#subprocess.run
```
If encoding or errors are specified, or text is true, file objects for stdin, stdout and stderr are opened in text mode using the specified encoding and errors or the io.TextIOWrapper default. The universal_newlines argument is equiva

//snip 

Changed in version 3.7: Added the text parameter, as a more understandable alias of universal_newlines. Added the capture_output parameter.
```

Reply:
updated.